testing/cpm/cpm_cuco-simple.cmake

```diff
@@ -0,0 +1,31 @@
+#=============================================================================
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#=============================================================================
+include(${rapids-cmake-dir}/cpm/init.cmake)
+include(${rapids-cmake-dir}/cpm/cuco.cmake)
+
+rapids_cpm_init()
+
+if(TARGET cuco::cuco)
+  message(FATAL_ERROR "Expected cuco::cuco expected to not exist")
```

Comment:
This error message doesn't make sense to me, but I might be reading the sentence wrong.

Reply:
The target shouldn't exist until after the first call to `rapids_cpm_cuco` below.

Reply:
Looks like there is a typo with an extra "expected" though. I'll fix.

Reply:
Yup. Raising the error makes sense -- the message just had an [unexpected expected](https://giphy.com/clips/ralph-dramatic-turn-hamster-2eeltIELxgHQ9Y5iA3).